src/main/java/jsr223/perl/PerlCommandCreator.java

```diff
@@ -38,6 +37,8 @@
 @Getter
 public class PerlCommandCreator {
 
+    private static final String PERL_COMMAND = "";
```

Comment:
It will not work like this. Before the perl_command by default was "perl". Why do you put empty string?
It should be PERL_COMMAND = "perl";
Have you tried to run compiled jar from this sources on your local?